code/modules/cciaa/cciaa_items.dm

```diff
@@ -1,5 +1,9 @@
 //CCIAA's tape recorder
 /obj/item/device/taperecorder/cciaa
+	name = "Human Resources Recorder"
+	desc = "A recorder modified for the specific use-cases of human resource departments around the galaxy."
+	desc_info = "This recorder records the fingerprints of the interviewee, to do so, interact with this recorder when asked."
```

Comment:
would place this under `desc_fluff` to be in line with how ordering is usually.

Reply:
desc info is to be used for mechanical descriptions.

Reply:
not what i said.